samples/compose-samples/build.gradle.kts

```diff
@@ -20,7 +20,10 @@ android {
 }
 
 dependencies {
+  val composeBom = platform(libs.androidx.compose.bom)
```

Comment:
You should be able to lose this and just have `implementation(platform(libs.androidx.compose.bom))`.  The `androidTestImplementation` configuration extends `debugImplementation`, which in turn extends `implementation`.

Reply:
oh weird, the Android docs specified to do it this way :S.

Reply:
oh weird, the Android docs specified to do it this way :S.